dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/rest/TelemetryClientFilter.java

```diff
@@ -71,6 +71,11 @@ public void init() {
             if (instrumenter != null) {
                 return;
             }
+            
+            if(openTelemetry.toString().equals(OpenTelemetry.noop().toString())){
+                this.instrumenter = null;
+                return;
+            }
```

Comment:
I don't think this works. If `instrumenter` is `null`, won't the `filter` methods throw a `NullPointerException`?

For `TelemetryClientFilter`, can we do the check in `TelemetryClientBuilderListener` and `TelemetryRestClientBuilderListener` to avoid adding the filter in the first place if telemetry is not enabled?